shared/src/models/utils/personnel-type.ts

```diff
@@ -1,21 +1,5 @@
-import type { AllowedValues } from '../../utils/validators/index.js';
+import type { ExerciseState } from '../../state.js';
 
-export type PersonnelType = 'gf' | 'notarzt' | 'notSan' | 'rettSan' | 'san';
```

Comment:
This type is still used in `shared/src/simulation/activities/reassign-treatments.spec.ts`